Formula/exa.rb

```diff
@@ -32,6 +32,11 @@ class Exa < Formula
   end
 
   def install
+    # Workaround rust 1.61.0 bug, remove with next rust release
+    # https://github.com/rust-lang/rust/issues/97255
+    # https://github.com/ogham/exa/issues/1068
+    ENV["CARGO_PROFILE_RELEASE_LTO"] = "false"
```

Comment:
Can we `raise` an error when the Rust version changes?

Reply:
good idea!